hello-world.java

```diff
@@ -0,0 +1 @@
+system.out.println("Hello World!");
```

Comment:
Hi, thanks for your contribution! However, this code seems not very valid. Can you please change that? Thanks :ok_hand: